src/shared/components/navbar/desktop/index.module.css

```diff
@@ -21,7 +21,8 @@
         display: flex;
         justify-content: center;
 
-        & a {
+        & a,
+        & .link {
             margin-right: 3%;
             display: flex;
             align-items: center;
```

Comment:
Indentation in the lines above is wrong I think.

Reply:
I don't think so. Lint is not complaining about it 🤔

Reply:
It was a GitHub display bug I guess.